src/encoded/schemas/quality_metric_chipseq_v2.json

```diff
@@ -0,0 +1,1362 @@
+                                    "qc_order": 9
+                                }
+                            }
+                        },
+                        "rep2": {
+                            "type": "object",
+                            "qc_order": 2,
+                            "additionalProperties": true,
+                            "properties": {
+                                "subsampled_reads": {
+                                    "title": "Subsampled reads",
+                                    "type": "number",
+                                    "qc_order": 1
+                                },
+                                "estimated_fragment_len": {
+                                    "title": "Estimated fragment length",
+                                    "type": "number",
+                                    "qc_order": 2
+                                },
+                                "corr_estimated_fragment_len": {
+                                    "title": "Estimated fragment length correlation",
+                                    "type": "number",
+                                    "qc_order": 3
+                                },
+                                "phantom_peak": {
+                                    "title": "Phantom peaks",
+                                    "type": "number",
+                                    "qc_order": 4
+                                },
+                                "corr_phantom_peak": {
+                                    "title": "Phantom peak correlation",
+                                    "type": "number",
+                                    "qc_order": 5
+                                },
+                                "argmin_corr": {
+                                    "title": "Argmin correlation",
+                                    "type": "number",
+                                    "qc_order": 6
+                                },
+                                "min_corr": {
+                                    "title": "Min correlation",
+                                    "type": "number",
+                                    "qc_order": 7
+                                },
+                                "NSC": {
+                                    "title": "Normalized strand cross-correlation coefficient",
+                                    "type": "number",
+                                    "qc_order": 8
+                                },
+                                "RSC": {
+                                    "title": "Relative strand cross-correlation coefficient",
+                                    "type": "number",
+                                    "qc_order": 9
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "jsd": {
```

Comment:
@clarabakker Could you check  `jsd` property since its location looks not clear to me. Is it a top-level QC field like `align_enrich` / `lib_complexity`, or is it a sub-property of `align_enrich`? If it is sub-property, it might be moved into the `properties`.

Reply:
This all looks great--thank you, Utku! You are right, jsd should be a property of of `align_enrich`.